docs/developer/style.md

```diff
@@ -0,0 +1,219 @@
+# Go Style Guide in Vald
+
+## Table of Contents
+
+- [Introduction](#Introduction)
+- [Style](#Style)
+- [Test](#Test)
+	- [Table-Driven-Test](#Table-Driven-Test)
+
+## Introduction
+
+## Style
+
+## Test
+
+Testing guideline has 2 important rules for the coding quality and readability
+1. Use Table-Driven-Test
+2. Keep code coverage over 85%
+   - test coverage != high testing quality, but low coverage means bad testing quality
```

Comment:
maybe we need a sample command for user to check the coverage? 
`go test -cover ./...`